flight/Libraries/math/coordinate_conversions.c

```diff
@@ -366,6 +366,75 @@ void rot_mult(float R[3][3], const float vec[3], float vec_out[3], bool transpos
 }
 
 /**
+ * @brief LLA_linearization_float calculates linearized NED to LLA conversion using floats
```

Comment:
Is this comment right? Isn't it the other way around, i.e. this is performing an LLA-->NED transformation linearization?